catroid/src/org/catrobat/catroid/utils/Utils.java

```diff
@@ -680,4 +680,8 @@ public static boolean isUserLoggedIn(Context context) {
 					|| token.equals(ServerCalls.TOKEN_CODE_INVALID));
 		return tokenValid;
 	}
+
+	public static String getNumberStringForBricks(float value) {
+		return (int) value == value ? "" + (int) value : "" + value;
```

Comment:
Please use somthing like value.intValue(); Why do you cast into an integer, when return is a String?